resources/default-config.kdl

```diff
@@ -459,6 +459,16 @@ binds {
     Ctrl+Print { screenshot-screen; }
     Alt+Print { screenshot-window; }
 
+    // Applications such as remote-desktop clients and software KVM switches may
+    // request that niri stops processing the keyboard shortcuts defined here
+    // so they may, for example, forward the key presses as-is to a remote machine.
+    // It's a good idea to bind an escape hatch to toggle the inhibitor,
+    // so a buggy application can't hold your session hostage.
+    //
+    // The allow-inhibiting=false property can be applied to other binds as well,
+    // which ensures niri always processes them, even when an inhibitor is active.
+    Mod+Shift+Escape allow-inhibiting=false { toggle-keyboard-shortcuts-inhibit; }
```

Comment:
I think we should bind this to `Mod+Escape` by default because Super+Escape is the default in GNOME: https://gitlab.gnome.org/GNOME/mutter/-/blob/d7d92c68bd5c76525752feab65c44b3157deb7f1/data/org.gnome.mutter.wayland.gschema.xml.in#L58-61

Reply:
The reasoning for *not* doing this is so:

The primary release mechanism should be implemented in the client. It will know best what keys are in use and easily accessible. In many cases, we'd expect the client to release on `Super+Escape` (because inhibiting shortcuts means you're free to use `Super` for whatever binds, and `Super+Escape` is unlikely to mean anything else), but if it does mean something else then the client knows best what is an appropriate release shortcut.

The reason why i chose `Mod+Shift+Escape` is because that's the shortcut to open Task Manager on Windows; so it already has the connotation of "An app is misbehaving!! Help me, operating system!" in some users' minds (including my own).

The difference is, however, ultimately arbitrary. So if you disagree with the above, i'll change it to `Mod+Escape`.